rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/TableMetaCache.java

```diff
@@ -90,6 +94,25 @@ public static TableMeta getTableMeta(final DataSourceProxy dataSourceProxy, fina
         return tmeta;
     }
 
+    /**
+     * Clear the table meta cache
+     * @param dataSourceProxy
+     */
+    public static void refresh(final DataSourceProxy dataSourceProxy){
+        ConcurrentMap<String, TableMeta> tableMetaMap = TABLE_META_CACHE.asMap();
+        for (Entry<String, TableMeta> entry : tableMetaMap.entrySet()) {
+            try {
+                TableMeta tableMeta = fetchSchema(dataSourceProxy, entry.getValue().getTableName());
+                if(tableMeta == null){
+                    LOGGER.error("get table meta error");
+                }
+                TABLE_META_CACHE.put(entry.getKey(), tableMeta);
```

Comment:
It is best to compare and implement the equal method. When the table structure does not change, but a new object is returned here.